ui/app/AppLayouts/Profile/helpers/ProfileShowcaseModelAdapter.qml

```diff
@@ -57,15 +57,12 @@ QObject {
             },
             FastExpressionRole {
                 name: "membersCount"
-                expression: model.members.count
-                expectedRoles: ["members"]
+                expression: model.allMembers.count
```

Comment:
```suggestion
                expression: model.allMembers.ModelCount.count
```

Reply:
I couldn't use `ModelCount` because allMembers doesn't come from a SFPM. Is there another another way to fix it?

Reply:
> I couldn't use `ModelCount` because allMembers doesn't come from a SFPM. Is there another another way to fix it?

Hmm, I'm not sure what you mean. `ModelCount`'s main benefit is it works with any model: models from nim, proxy models, ListModel and so on.

Reply:
Rather `model.allMembers.rowCount()`?